contexts/alumni/v2.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@version": 1.1,
```

Comment:
```suggestion
```

Reply:
Fixed in edited/updated [8998f2d](https://github.com/credential-handler/vc-examples/pull/76/commits/8998f2d973051d9e14c03c9a69f0be6dffaf708b)